judgehost.md

```diff
@@ -49,13 +51,13 @@ make judgehost && sudo make install-judgehost
 ### 添加用户
 
 ```shell
-useradd -d /nonexistent -U -M -s /bin/false domjudge-run
-# 如果 judgehost 拥有多个 CPU 核心，你可以添加额外的用户来支持绑定
+sudo useradd -d /nonexistent -U -M -s /bin/false domjudge-run
+# 如果 judgehost 拥有多个 CPU 核心，你可以添加额外的用户来支持绑定，但是要记得修改 judgehost-static.php 等文件里的 `RUNUSER` 。
```

Comment:
Why modify `RUNUSER`?

Reply:
My apologies. I was misunderstood the original author`s meaning. It is a WRONG step. I have already fix it in the newest PR. Thank you for your thoughtfulness.